pages/go.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 常用链接
+layout: page
+permalink: /go/
+---
+<hr style="border:2px solid gray">
+
+### **网站** 
+* [社区主页](https://www.zgzg.io/)
+* [购票](https://www.zgzg.io/singerticket/)
+
+### **志愿者相关**
+
+* [全体志愿者名单及联系方式](https://zgzg.link/zgid-public)
+* 各公司[配捐指南](https://zgzg.link/log-instruction)
+
+### **开发者相关**
+
+* 视频教程：[如何创建 zgzg.link 短链接](https://youtu.be/8HrtHwGEOoE)
+* [网站源代码Github](https://github.com/zgzgorg/yj.zgzg.io)
+
+<img id="yj-img" src="https://res.cloudinary.com/zaigezaigu/image/upload/v1666995496/zgzg-io-website/link_djn1i3.png" style="width:600px;height:200px;" />
+
```

Comment:
1. First-level heading should be first-level, not third-level.
2. Headings are emphasized adequately. Emphasis (`**...**`) shouldn't be needed.
3. Good to see that you know how to add an image. I was hoping to see `![](...)`, though. I removed it for the sake of faster page load time -- Remember, this page is for quick navigation of common links, so speed it priority.

```suggestion

# 网站
* [社区主页](https://www.zgzg.io/)
* [购票](https://www.zgzg.io/singerticket/)

# 志愿者相关

* [全体志愿者名单及联系方式](https://zgzg.link/zgid-public)
* 各公司[配捐指南](https://zgzg.link/log-instruction)

# 开发者相关

* 视频教程：[如何创建 zgzg.link 短链接](https://youtu.be/8HrtHwGEOoE)
* [网站源代码](https://github.com/zgzgorg/yj.zgzg.io)

```